### ISSUE TITLE
Data order when using columns.data as object

### ISSUE THREAD

**Issue by rockshandy** [DataTables/ColReorder#35]
I'm having an issue using ColReorder with column data defined as an object, but using a function for one type of value. For instance I have something like:

```javascript
columns: [
{
  data: {
    _: function(row, type, val, meta) {
      // return some fancy default
    }
  },
  {
    data: {
      _: 'propName'
    }
  }
]
```

From the data tables api, I believe functions in this case are okay, and my table displays the data I expect. Column reorder seems to error in the first column case when executing the "Update the Get and Set functions for each column" loop where column data is an object literal. Is this intended?

**Comment by DataTables** [comment]
Can you link to a test page showing the issue please.

**Comment by rockshandy** [comment]
It will take me a little time to make up an example, but I am willing to do that. I've seen other people easily putting something up on a jsfiddle, is there some standard way people often post a test page?

**Comment by DataTables** [comment]
JSFiddle or http://live.datatables.net which is setup a little be more for the DataTables libraries. All of the libraries are also available on the [DataTables CDN](//cdn.datatables.net) for quick inclusion in JSFiddle etc.

Thanks!

**Comment by rockshandy** [comment]
I think this example show what I mean: http://live.datatables.net/siruboki/1/edit

I took some of the data already there and converted it to an object. I use that as my table data and set various ways to get data from the object for each column. In office, I tried to replicate my case where data is described in an  object for that column which displays fine. Once I do this, I can't reorder my columns. Changing that column setting to the name 'office' makes it work again.

Does that jsfiddle help?
